### ISSUE TITLE
Explain a bit more what Ruru is?

### ISSUE THREAD

**Issue by sebastiancarlos** [graphql/graphql.github.io#1625]
The page https://graphql.org/graphql-js/running-an-express-graphql-server/ introduces the Ruru dependency. Yet, it's not really explained. Later in the page, it is referred to as "Graph_i_QL", which is not its name but rather the name (I suppose) of the previous tool used in this documentation. 

By looking around, it seems that Ruru is [pre-release](https://github.com/graphile/crystal/tree/main/grafast/ruru), and there's [not much information](https://grafast.org/ruru) about it online.

Giving this situation, would you consider updating the page to explain a bit more what Ruru is, and why was it picked over GraphiQL?

**Comment by benjie** [comment]
Good call. Copying this from Reddit:

> Hi, Ruru author here, and also contributor to GraphiQL. Ruru is a very thin wrapper around GraphiQL which adds a couple extensions and makes it easy to get started by giving you a string you can serve easily. GraphiQL used to be bundled with express-graphql, but that library is now deprecated in favour of graphql-http, which has declared bundling GraphiQL to be out of scope (a view I support). Since there’s not currently a super simple way to get GraphiQL itself running with graphql-http, we’re (temporarily?) recommending Ruru until we come up with a cleaner solution. Ruru itself was put together to go with PostGraphile and Grafast (a new execution engine for GraphQL) but I hope it will be useful beyond that.
> 
> TL;DR: Ruru is a wrapper around GraphiQL

**Comment by benjie** [comment]
Addressed in #1699
